news/_posts/2020-03-12-ps17-minor-release-lifecycle.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title: "PrestaShop 1.7 minor release lifecycle"
+subtitle: "The journey of a minor release of PS 1.7, step by step"
+date:   2020-03-12 16:00:00
```

Comment:
```suggestion
date:   2020-03-24 14:00:00
```
@matks ok for you? can I merge it after updating the date?

Reply:
Ok for me 👍